Zotero/Scenes/General/Views/ReaderSettingsViewController.swift

```diff
@@ -56,6 +61,20 @@ final class ReaderSettingsViewController: UICollectionViewController {
             .disposed(by: disposeBag)
     }
 
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        updatePreferredContentSizeIfNeeded()
+
+        func updatePreferredContentSizeIfNeeded() {
+            let contentHeight = max(collectionView.collectionViewLayout.collectionViewContentSize.height, minimumPreferredContentSize.height)
+            let contentWidth = max(collectionView.collectionViewLayout.collectionViewContentSize.width, minimumPreferredContentSize.width)
```

Comment:
I'd keep the width fixed, collection height should be calculated based on the fixed width and the preferred width will always be bigger anyway.